test/ex_plasma/typed_data/transaction_test.exs

```diff
@@ -29,11 +29,11 @@ defmodule ExPlasma.TypedData.TransactionTest do
 
   describe "hash/1" do
     test "hashes a eip 712 encoded object" do
-      encoded_hash = ExPlasma.TypedData.hash(%Transaction{})
+      encoded_hash = ExPlasma.TypedData.hash(%PaymentV1{})
 
       assert encoded_hash ==
-               <<237, 222, 56, 55, 149, 76, 223, 131, 240, 226, 246, 122, 166, 114, 38, 102, 122, 183, 230, 80, 135,
-                 114, 118, 119, 47, 205, 121, 140, 23, 172, 117, 213>>
+               <<196, 145, 245, 73, 70, 135, 10, 204, 85, 216, 199, 89, 153, 191, 31, 94, 60, 22, 20, 81, 54, 74, 38,
```

Comment:
some tests values were changed. is it ok?

Reply:
Yep, I had a comment in the original PR, this is due to the fact that the original code was using tx_type=0, I changed it to 1.